src/reducers/root-reducer.js

```diff
@@ -0,0 +1,4 @@
+var changeFixture = require('./change-fixture.js');
+
+// only exports one reducer now, please use combineReducers when more are added
```

Comment:
Comments are sentences therefore they should be formatted accordingly. Please capitalize the first word and add a period at the end of the sentence.